compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/optimizer/BIRLockOptimizer.java

```diff
@@ -0,0 +1,258 @@
+/*
+ *  *  Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
This formating looks weird. Can you check this?

Reply:
Will check and update

Reply:
Fixed